cves/CVE-2017-12611.yml

```diff
@@ -142,10 +148,13 @@ interesting_commits:
     emerging themes?
     If there are no interesting commits, demonstrate that you completed this section
     by explaining what happened between the VCCs and the fix.
-  answer:
+  answer: |
+    This is a great example of a commit where the user is just performing maintenence
+    Cleaning up imports at the same time as fixing bugs. It is commits like this that
+    let code be maintained years later without vulnerabilities like this.
```

Comment:
Were there many commits like this? Was this one of many clean-up commits during a whole code-base clean-up or was this a single clean-up added onto a bugfix?

Reply:
Added some commentary about the lack of commits surrounding this file. It had barely been touched since its initial creation.